### PR TITLE
Added more triggers for Shruggie goodie

### PR DESCRIPTION
**Pull request by 0xbsec**
Added more triggers for Shruggie goodie.
shruggie

---
IA Page: https://duck.co/ia/view/shruggie

### CODE REVIEW

File: lib/DDG/Goodie/Shruggie.pm [hunk 1]
```diff
@@ -13,32 +13,32 @@ use strict;
...
 use JSON; # encoded the various other ASCII items in a JSON file
 
-
-zci answer_type => "shruggie";
+zci answer_type => 'shruggie';
 zci is_cached   => 1;
 
 # Triggers
-triggers start => "shruggie";
+triggers any => 'shruggie','ascii emoji','ascii emoji art','emoji','emojicon','emoticon','emojiton','kaomoji','face mark';
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/Shruggie.pm]: `emojiton`?

Comment (0xbsec) [inline lib/DDG/Goodie/Shruggie.pm]: Sorry `emojiton` was added by mistake. I've discussed with @moollaza about adding more triggers, please take any action you see necessary.

Comment (GuiltyDolphin) [inline lib/DDG/Goodie/Shruggie.pm]: I would say pull the triggers out into either a separate list or file, then remove the unnecessary triggers (such as `emojiton`).

File: t/Shruggie.t [hunk 2]
```diff
@@ -78,16 +88,46 @@ ddg_goodie_test(
     'shruggie AND hug me' => test_zci('Shruggie and Hug Me',
             structured_answer => build_structure('Shruggie and Hug Me',{
                 "Shruggie" => '¯\_(ツ)_/¯',
-                "Hug Me" => "(っ◕‿◕)っ",
+                "Hug Me" => "(っ◕‿◕)っ"
```
Comment (GuiltyDolphin) [inline t/Shruggie.t]: I know it's personal preference (mostly) but I'd prefer keeping the trailing commas - it also makes the diff easier to read.

**Comment by daxtheduck** [comment]
## [Shruggie & Friends](https://duck.co/ia/view/shruggie)

**Description**: Provides ASCII variations of the "shruggie" ( i.e. ¯\_(ツ)_/¯ ) that I can be copy/pasted.

Also provides a list of other "friends" for the query "shruggie and friends".

**Example Query**: [shruggie and friends](https://beta.duckduckgo.com/?q=shruggie and friends), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**:  

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/shruggie)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/shruggie)*

File: t/Shruggie.t [hunk 3]
```diff
@@ -46,10 +46,20 @@ ddg_goodie_test(
             name => 'Shruggie',
             description => 'Emojii for everone',
             templates => {
-                group => 'text'
+                group => 'list',
+                options => {
+                    content => 'record'
+                }
+
```
Comment (GuiltyDolphin) [inline t/Shruggie.t]: Lets lose this extra line.

File: t/Shruggie.t [hunk 4]
```diff
@@ -78,16 +88,46 @@ ddg_goodie_test(
...
+            },
+            ["Table flip"]
+        )),
     # Try to include some examples of queries on which it might
     # appear that your answer will trigger, but does not.
     'shruggie and' => undef,
     'shruggie and friendss' => undef,
-    'shruggie andover' => undef
+    'shruggie andover' => undef,
+    'ascii table flip' => undef
```
Comment (GuiltyDolphin) [inline t/Shruggie.t]: Lets get a few checks to ensure things like `tableflip` don't trigger it.

**Comment by moollaza** [comment]
@0xbsec Ping! I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

**Comment by moollaza** [comment]
@0xbsec thanks again for improving the Shruggie Goodie! You’ve been doing a really good job here and we'd like you to help our community tackle the next big challenge - our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements 

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

**Comment by GuiltyDolphin** [comment]
@0xbsec I'm closing this for now - if you want to finish it up later just let me know and we can re-open it.

**Pull request closed without merging**